pages/common/unzip.md

```diff
@@ -13,3 +13,7 @@
 - List the contents of a zip file without extracting:
 
 `unzip -l {{file}}`
+
+- Extract the contents of a zip file to the console:
+
+`unzip -c {{file}}`
```

Comment:
While we are at it, please rename {{file}} to {{file.zip}} everywhere for clarity.

Reply:
I had thought about that. This page does use `compressed.zip`, but only in the extract case where you need to mention 2 files - compressed_file and extracted_file. If 2 files are not needed, then I think for simplicity's sake, just mentioning `file.zip` is good enough.